tests/gnrc_dhcpv6_client_ia_na/tests-as-root/01-run.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2018 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import pexpect
+import sys
+import time
+
+from scapy.all import AsyncSniffer, sendp, Ether, IPv6, UDP
+from scapy.all import DHCP6_Solicit, DHCP6_Advertise, DHCP6_Request, DHCP6_Reply
+from scapy.all import DHCP6OptClientId, DHCP6OptServerId, DHCP6OptMudUrl
```

Comment:
This should be updated to the new mechanism for the MUD option introduced in #16593. Then again: This test is primarily for the IA_NA, so alternatively you could just remove all the MUD option tests as they are not significant for this test.

Reply:
I was a bit unsure in general about adding a separate test as it causes a lot of code duplication. Maybe the updated test in `tests/gnrc_dhcpv6_client` is enough? It does not cover the auto initialization, though, but maybe this isn't necessary?

Reply:
I would prefer if functionalities are tested separately as faults can then be easily pin-pointed. However, this increases build times and currently these tests are not run automatically since they require root privileges. So just updating `tests/gnrc_dhcpv6_client` might be preferable. You do not add changes to auto-init anyway, right?

To address my first concern of pin-pointing faults, maybe the IA_NA tests can be marked in the output specially with an assert message?

Reply:
> You do not add changes to auto-init anyway, right?

I added [this](https://github.com/RIOT-OS/RIOT/pull/16228/files#diff-892a732ccdabad2f85826ca80908ed7922efdf2387f52f8ef8707b0835484aeeL122-R140)  to the `thread` function called by `dhcpv6_client_auto_init` but you are right, the other aspects of the auto-init haven't been touched.

> To address my first concern of pin-pointing faults, maybe the IA_NA tests can be marked in the output specially with an assert message?

Oh, I noticed only now that there is an assertion missing for the `IA_NA` part! I will add extend the test and then the new functionality should hopefully be covered :)

Reply:
I added now an assertion to the `gnrc_dhcpv6_client` tests (maybe the wording could be improved a bit) and remove the separate test directory.